src/modules/esl-footnotes/core/esl-footnotes.ts

```diff
@@ -146,13 +161,17 @@ export class ESLFootnotes extends ESLBaseElement {
     });
   }
 
-  public turnOffHighlight(note: ESLNote) {
+  /** Turn off highlight from notes with the same text */
+  public turnOffHighlight(note: ESLNote): void {
     this._notes
       .filter((item) => note.html === item.html)
       .forEach((item) => item.highlight(false));
   }
 
-  protected _sendRequestToNote() {
+  /**
+   * Send a request to all notes, expecting to get a response from
+   * the unlinked ones and link up with them */
+  protected _sendRequestToNote(): void {
```

Comment:
Just an idea for naming: `_notifyNotes()`

Reply:
That's a great proposition.